src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorMenuItem.class.php

```diff
@@ -66,13 +66,13 @@ class TCMSTableEditorMenuItem
 
     /**
      * bootstrap button style (default: btn-primary)
-     * possible values in bootstrap 3.0: btn-default, btn-primary, btn-success, btn-info, btn-warning, btn-danger, btn-link.
+     * possible values in bootstrap 4.1: btn-secondary, btn-primary, btn-success, btn-info, btn-warning, btn-danger, btn-link.
```

Comment:
Some OCD thing, but it hurts to read "secondary" before "primary" .. :-)

Reply:
This was a search&replace... changed the position.